Cargo.toml

```diff
@@ -19,6 +19,7 @@ parking_lot = "0.12"
 pin-project = "1"
 tokio = { version = "1", features = ["rt"] }
 tracing = "0.1"
+colored = "2"
```

Comment:
Maybe we should hide this under a feature?

Reply:
Both LGTM :)

Reply:
Yeah, both LGTM. They are about the balance between performance and convenience.